### DISCUSSION TITLE
Scalable RAG with CosmosDB for NoSQL: Session Resources

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#41] [Announcements]
This is the resources and discussion thread for this live stream: https://aka.ms/raghack/cosmosdb/nosql
It will be updated with links after the session.

Recording: https://www.youtube.com/watch?v=YZvNaYc_BLM
Slides: https://github.com/microsoft/RAG_Hack/blob/main/slides/RAGHACKCosmosDBNoSQL.pptx

Please ask any follow-up questions here, and the speakers will see them.

**Comment by jcodella** [comment]
Hey folks, thanks for tuning in today. If you have any questions about Azure Cosmos DB for NoSQL and vector search, RAG, or building AI apps in general, please reach out to us!: [CDB4AI@microsoft.com](mailto:CDB4AI@microsoft.com)
